tap_chargebee/streams/base.py

```diff
@@ -199,9 +197,16 @@ def sync_data(self):
         else:
             params = {"updated_at[after]": bookmark_date_posix, "updated_at[before]": self.START_TIMESTAP}
             bookmark_key = 'updated_at'
-
+        
         LOGGER.info("Querying {} starting at {}".format(table, bookmark_date))
 
+        if self.config.get("filters"):
+            entity_filters: dict[str,str] = self.config.get("filters", {}).get(self.ENTITY)
+            if entity_filters:
```

Comment:
If you define `entity_filters` as an empty dict you would be able to make this if block one indentation level lower than what it's now